docs tidy-up (#602)
ACCESS-NRI/access-om3-configs: documentation/docs/index.md

# Home

Welcome to the documentation for the [ACCESS-OM3 ice-ocean model configurations](https://github.com/ACCESS-NRI/access-om3-configs)!

See the links on the left, particularly the [quick start](pages/Quick-start.md).

## Contributions welcome 🙏
This documentation is a work in progress, and we welcome any contributions, including corrections and suggestions.

Contributions can be made in a few ways.

## Report a bug or make a suggestion
Issues should be posted to [github.com/ACCESS-NRI/access-om3-configs/issues](https://github.com/ACCESS-NRI/access-om3-configs/issues). All contributions are welcome but we would particularly appreciate text suggestions (below).

> ## Quick contributions 

Author: ```suggestion
### Quick contributions 
```

!!! tip
    For external people, this method does not allow one to preview the changes rendered into a website or create whole new pages. Its advantage is that it's *very quick* to do.<br>

The simplest and fastest way to make a change to an _existing_ page is to click the edit "pencil" on the top-right corner. This will go to the relevant GitHub markdown file and clicking the top-right pencil again on GitHub will allow you to edit the file. Once complete, click `Commit changes...`. There are then _two_ possibilities, depending on whether you have  write access to [`access-om3-configs`](https://github.com/ACCESS-NRI/access-om3-configs): 

1.  **No write access** (e.g. you are not part of the `ACCESS-NRI` GitHub organisation): this will prompt you to make a fork and then a pull request (less than 1 minute!). 
1.  **You have write access**: please commit changes on a new branch and then use a pull request (this relates to the next option). 

> ## Larger contributions (online PR-previews)

Author: ```suggestion
### Larger contributions (online PR-previews)
```

!!! tip

>     This method allows you create whole new pages, and to preview the changes rendered into a website. It does not require you to install any software.
>     Only for people with write access to [`access-om3-configs`](https://github.com/ACCESS-NRI/access-om3-configs).<br>

Author: ```suggestion
    This method allows you create whole new pages, and to preview the changes rendered into a website. It does not require you to install any software, but is **only available for people with write access to [`access-om3-configs`](https://github.com/ACCESS-NRI/access-om3-configs)**.<br>
```

> Create a new branch, e.g. `jblogs/doc-update`, make doc changes (the documentation sources are in [github.com/ACCESS-NRI/access-om3-configs/tree/main/documentation](https://github.com/ACCESS-NRI/access-om3-configs/tree/main/documentation)), then open a related PR and a GitHub preview will be made automatically.

Author: I don't understand your point, but correct it if you want.

> Create a new branch, e.g. `jblogs/doc-update`, make doc changes (the documentation sources are in [github.com/ACCESS-NRI/access-om3-configs/tree/main/documentation](https://github.com/ACCESS-NRI/access-om3-configs/tree/main/documentation)), then open a related PR and a GitHub preview will be made automatically.

Reviewer: For example, click the pencil on:
 - https://access-om3-configs.access-hive.org.au/pr-preview-602/pages/configurations/Overview/
 - https://access-om3-configs.access-hive.org.au/pr-preview-602/
Due to the stubs magic, the configurations pages won't be built off the main branch. 

I think this needs @atteggiani's wizardry though, so let's just leave it for now.

> ## Larger contributions (`mkdocs` offline)

Author: ```suggestion
### Larger contributions (`mkdocs` offline)
```

# Home

Welcome to the documentation for the [ACCESS-OM3 global ice-ocean model configurations](https://github.com/ACCESS-NRI/access-om3-configs)!

See the links on the left, particularly the [quick start](pages/Quick-start.md).

## Contributions welcome 🙏
This documentation is a work in progress, and we welcome any contributions, including corrections and suggestions.

Contributions can be made in a few ways.

## Report a bug or make a suggestion

> Issues should be posted to [github.com/ACCESS-NRI/access-om3-configs/issues](https://github.com/ACCESS-NRI/access-om3-configs/issues). All contributions are welcome but we would particularly apprecaite text suggestions (below).

Author: ```suggestion
Issues should be posted to [github.com/ACCESS-NRI/access-om3-configs/issues](https://github.com/ACCESS-NRI/access-om3-configs/issues). All contributions are welcome but we would particularly appreciate text suggestions (below).
```

## Quick contributions 
!!! tip

>     For external people, this method does not allow one to preview the changes rendered into a website or create whole new pages. It's advantage is that it's *very quick* to do.<br>

Author: ```suggestion
    For external people, this method does not allow one to preview the changes rendered into a website or create whole new pages. Its advantage is that it's *very quick* to do.<br>
```

    For people with write and without write access to `ACCESS-OM3-CONFIGS`.<br>

> If a change is being made to an existing page, then the simplest and fastest way to make a change is to click the edit "pencil" on the top-right corner of each rendered page. This will go to the relevant GitHub markdown file and clicking the top-right pencil again on GitHub will allow one to edit the file. Once complete click `Commit changes...`. There are then _two_ possibilities, depending on whether you have  write access to `ACCESS-OM3-CONFIGS`: 

Author: ```suggestion
The simplest and fastest way to make a change to an _existing_ page is to click the edit "pencil" on the top-right corner. This will go to the relevant GitHub markdown file and clicking the top-right pencil again on GitHub will allow you to edit the file. Once complete, click `Commit changes...`. There are then _two_ possibilities, depending on whether you have  write access to [`access-om3-configs`](https://github.com/ACCESS-NRI/access-om3-configs): 
```

1.  **No write access** (e.g. you are not part of the `ACCESS-NRI` GitHub organisation): this will prompt one to make a fork and then pull request (less than 1 minute!). 
1.  **You have write access**: please commit changes on a new branch and then use a pull request (this relates to the next option). 

## Larger contributions using online PR-previews
!!! tip

>     This method does allow one to preview the changes rendered into a website and create whole new pages. It does not require one to install any software.<br>

Author: ```suggestion
    This method allows you create whole new pages, and to preview the changes rendered into a website. It does not require you to install any software.<br>
```

>     Only for people with write access to `ACCESS-OM3-CONFIGS`.<br>

Author: ```suggestion
    Only for people with write access to [`access-om3-configs`](https://github.com/ACCESS-NRI/access-om3-configs).<br>
```

Create a new branch, e.g. `jblogs/doc-update` make doc changes (the documentation sources are in [github.com/ACCESS-NRI/access-om3-configs/tree/main/documentation](https://github.com/ACCESS-NRI/access-om3-configs/tree/main/documentation)), then open a related PR and a GitHub preview will be made automatically.

## Larger contributions (offline)
!!! tip

>     This method does allow one to preview the changes rendered into a website and create whole new pages. It requires one to install mkdocs and takes the longest to set up.<br>
>     For people with write and without write access to `ACCESS-OM3-CONFIGS`.<br>

Author: ```suggestion
    Similar to the above, this method allows you create whole new pages and to preview the changes rendered into a website. It works whether or not you have write access to [`access-om3-configs`](https://github.com/ACCESS-NRI/access-om3-configs), but requires you to install `mkdocs` and takes the longest to set up.<br>
```
